### PR TITLE
Ng16 migration

### PR DESCRIPTION
**Pull request by VitoAlbano**
**Please check if the PR fulfills these requirements**

> - [ ] The commit message follows our [guidelines](https://github.com/Alfresco/alfresco-ng2-components/wiki/Commit-format)
> - [ ] Tests for the changes have been added (for bug fixes / features)
> - [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")

> - [ ] Bugfix
> - [ ] Feature
> - [ ] Code style update (formatting, local variables)
> - [ ] Refactoring (no functional changes, no api changes)
> - [ ] Build related changes
> - [ ] Documentation
> - [ ] Other... Please describe:

**What is the current behaviour?** (You can also link to an open issue here)

**What is the new behaviour?**

**Does this PR introduce a breaking change?** (check one with "x")

> - [ ] Yes
> - [ ] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

**Review by wojd0 — APPROVED** [review]

File: .github/workflows/release-branch.yml [hunk 1]
```diff
@@ -22,20 +22,57 @@ env:
   GH_RUN_NUMBER: ${{ github.run_attempt }}
   GH_BUILD_NUMBER: ${{ github.run_id }}
   JOB_ID: ${{ github.run_id }}
+  PROXY_HOST_BPM: ${{ secrets.E2E_HOST }}
```
Comment (DenysVuika) [inline .github/workflows/release-branch.yml:25]: I think it's incorrect rebase, those fields were cleaned up from the ADF recently

File: e2e/resources/adf/allFileTypes/documents/text/a_vsd_file.vsd
[the remark this answers is not in the document]
Comment (VitoAlbano) [reply e2e/resources/adf/allFileTypes/documents/text/a_vsd_file.vsd:1]: To be honest no, but it's a good catch i'll revert these changes and check again

File: .github/workflows/release-branch.yml (hunk 1 above)
Comment (VitoAlbano) [reply .github/workflows/release-branch.yml:25]: This was cleaned recently and in the conflicts between the two as i've fixed the command to match nx 16 they sneaked in, i'll remove them

File: package.json [hunk 2]
```diff
@@ -1,15 +1,36 @@
...
+    "00": "echo -------------------------------------------- DOC  -----------------------------------------------",
     "build-doc-tools": "tsc -p ./tools/doc/tsconfig.json",
     "docbuild": "(cd ./tools/doc; npm i) && node tools/doc/node_modules/typedoc/bin/typedoc --tsconfig lib/tsconfig.doc.json && node tools/doc/buildYamlSourceInfo.js docs/docs.json && node ./tools/doc/docProcessor.js",
+    "01": "echo -------------------------------------------- Build Lib  -----------------------------------------------",
     "affected:libs": "nx affected:libs",
+    "coverage": "./lib/config/create-coverage-index.sh && lite-server -c ./lib/config/proxy-coverage.json",
+    "03": "echo -------------------------------------------- Lint  -----------------------------------------------",
     "affected:lint": "nx affected:lint",
-    "clean": "rimraf dist node_modules dist/libs"
+    "04": "echo -------------------------------------------- Demo Shell -----------------------------------------------",
```
Comment (DenysVuika) [inline package.json:18]: incorrect rebase as this stuff is removed on develop

File: package.json [hunk 3]
```diff
@@ -29,82 +50,86 @@
...
     "apollo-angular": "^5.0.2",
-    "chart.js": "^4.3.0",
+    "chart.js": "4.4.4",
     "cropperjs": "1.6.2",
     "date-fns": "^2.30.0",
     "dotenv-expand": "^5.1.0",
     "event-emitter": "^0.3.5",
     "material-icons": "^1.13.12",
     "minimatch-browser": "1.0.0",
+    "monaco-editor": "0.41.0",
```
Comment (DenysVuika) [inline package.json:80]: was removed on develop

File: lib/core/src/lib/layout/components/layout-container/layout-container.component.scss [hunk 4]
```diff
@@ -31,7 +32,7 @@ adf-layout-container {
     overflow: hidden;
 }
 
-/* stylelint-disable selector-class-pattern */
+/* stylelint-disable selector-class-pattern, declaration-no-important */
 #{$mat-sidenav-content},
 #{$mat-drawer-transition} #{$mat-drawer-content} {
```
Comment (VitoAlbano) [inline lib/core/src/lib/layout/components/layout-container/layout-container.component.scss:37]: Do we need this rule?
Can't we get the same results without touching material selectors?

**Pull request merged**